### DISCUSSION TITLE
how to generate a runner_registration_token

### DISCUSSION THREAD

**Discussion by dderemiah** [semaphoreui/semaphore#1822] [Q&A]
Is anyone out there using semaphore runners?  If so, how did you generate the runner_registration_token mentioned in the documentation?

**Comment by canob** [comment]
Hi @dderemiah ,
Here you can see how to generate a token with CURL:
https://docs.semui.co/administration-guide/api#how-to-use-the-semaphore-api

**Reply by tboerger** [reply]
These tokens are not used for the runner registration.

The registration token for the to be defined within the server config and it got to match the token used to start a runner which can be any random string.

**Reply by canob** [reply]
Ok, thanks @tboerger, I misunderstood the documentation.

**Comment by tboerger** [comment]
You can use any password as a token
